packages/studio-plugin/tests/parsers/StaticParsingHelpers.test.ts

```diff
@@ -201,6 +202,23 @@ describe("parseJsxAttributes", () => {
     };
     expect(receivedPropValues).toEqual(expectedPropValues);
   });
+
+  it("parsing an negative numeric literal", () => {
```

Comment:
nit: can this test's title match the style of our other ones (verb phrase so that if you start with "it", they usually form a grammatically correct statement)? also super nit: `an` -> `a`